### ISSUE TITLE
Zoom auto de la carte

### ISSUE THREAD

**Issue by ydufresne-alesagglo** [OpenAgenda/wordpress#44]
Bonjour,
Dans la partie filtrage des événements sont présents, entre autres : le calendrier et la carte. Lorsque je filtre entre deux dates, la zoom de la carte s'ajuste et tout le planisphère s'affiche (mode zoom out max).
Cordialement,

**Comment by kaore** [comment]
Bonjour,

Le zoom out ce produit que lorsqu'aucun événement n'a été trouvé à priori. Si non, arrivez-vous à reproduire le problème directement sur la page de l'agenda sur https://openagenda.com ?

**Comment by ydufresne-alesagglo** [comment]
non, le problème ne se produit pas sur openagenda.com, uniquement sous WordPress en utilisant le plugin.
l'affichage de la carte est provoqué pas un do_shortcode dans un fichier de template.
je constate que si je place le do_shortcode dans event-loop.php le problème de zoom apparait lorsque l'on filtre, mais cela fonctionne correctement si je le place dans list-header.php

**Comment by vincedubroeucq** [comment]
Bonjour,
Le souci vient du fait que votre filtre est probablement à l'intérieur du conteneur rafraîchi quand les événements sont mis à jour.
La carte doit être placé en dehors du template event-loop.php.
Cordialement
